tests/rptest/services/redpanda.py

```diff
@@ -2891,14 +2891,16 @@ def remove_local_data(self, node):
 
     def redpanda_pid(self, node):
         try:
-            cmd = "pgrep --exact redpanda"
-
-            for p in node.account.ssh_capture(cmd,
-                                              allow_fail=True,
-                                              callback=int,
-                                              timeout_sec=10):
-                return p
-
+            cmd = "pgrep --list-full --exact redpanda"
+            for line in node.account.ssh_capture(cmd,
+                                                 allow_fail=True,
+                                                 callback=int,
+                                                 timeout_sec=10):
+                if "--version" in line:
```

Comment:
nit: a comment why the need for this check, like "redpanda --version is used by [...] and should not be considered for pid retrieval"

Reply:
Good point will add, do you know what it's used by? :)

Reply:
Okay added a comment. Good suggestion!